packages/cozy-pouch-link/src/helpers.js

```diff
@@ -1,13 +1,34 @@
 import startsWith from 'lodash/startsWith'
 
-export const withoutDesignDocuments = res => {
```

Comment:
Did you make sure that it was not used anywhere anymore?

Reply:
It's only use on `cozy-client` https://github.com/search?q=org%3Acozy+withoutDesignDocuments&type=Code